pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -212,6 +212,14 @@ func (db *DB) Migration() *migrate.Migration {
 					`UPDATE ttl SET expires = 1553727600 WHERE created <= 1553727600 `,
 				},
 			},
+			{
+				Description: "delete obsolete pieces",
+				Version:     5,
+				Action: migrate.Func(func(log *zap.Logger, mdb migrate.DB, tx *sql.Tx) error {
+					path := db.dbPath
```

Comment:
`db.dbPath` is not a directory, its DB file like `/home/wywrzal/.local/share/storj/local-network/storagenode/6/storage/piecestore.db`. Also `storage` directory currently contains more things than just old pieces (see image, old pieces directory is existing on my screenshot but old storage nodes will contain it). With this code you will remove everything.

![image](https://user-images.githubusercontent.com/7313031/55559018-99dac500-56ed-11e9-9e9b-6aa1b6c00ca1.png)

Reply:
it takes the directory where the dbpath is stored. Merging master, I  missed out the length of the directory name to be 2. 
`err == nil && info.IsDir() && len(f.Name()) == 2` the last condition looks for a directory of 2 chars length and removes all